opm/simulators/wells/VFPHelpers.cpp

```diff
@@ -648,14 +648,15 @@ INSTANCE(DenseAd::Evaluation<double, -1, 7u>)
 INSTANCE(DenseAd::Evaluation<double, -1, 8u>)
 INSTANCE(DenseAd::Evaluation<double, -1, 9u>)
 INSTANCE(DenseAd::Evaluation<double, -1, 10u>)
+INSTANCE(DenseAd::Evaluation<double, -1, 11u>)
 INSTANCE(DenseAd::Evaluation<double, 3, 0u>)
 INSTANCE(DenseAd::Evaluation<double, 4, 0u>)
 INSTANCE(DenseAd::Evaluation<double, 5, 0u>)
 INSTANCE(DenseAd::Evaluation<double, 6, 0u>)
 INSTANCE(DenseAd::Evaluation<double, 7, 0u>)
 INSTANCE(DenseAd::Evaluation<double, 8, 0u>)
 INSTANCE(DenseAd::Evaluation<double, 9, 0u>)
-
+INSTANCE(DenseAd::Evaluation<double, 10, 0u>)
```

Comment:
maybe also the part related to `INSTANCE(DenseAd::Evaluation`. I have to say that I am really not familiar with these. @akva2  thanks.

Reply:
it simply have evaluations with one more component than what was used before, so we need to instance for those evaluations. it's all good.